components/layout_2020/flow/inline.rs

```diff
@@ -1467,21 +1467,26 @@ impl InlineFormattingContext {
             }
 
             fn add_length(&mut self, l: Length) {
-                self.current_line.min_content += l;
-                self.current_line.max_content += l;
+                self.current_line.min_content += l.into();
+                self.current_line.max_content += l.into();
             }
 
             fn line_break_opportunity(&mut self) {
-                self.paragraph
-                    .min_content
-                    .max_assign(take(&mut self.current_line.min_content));
+                let max_min_content = std::cmp::max(
+                    self.paragraph.min_content,
+                    take(&mut self.current_line.min_content),
+                );
+                self.paragraph.min_content = max_min_content;
```

Comment:
is `max_min_content` needed?

```suggestion
                self.paragraph.min_content = std::cmp::max(
                    self.paragraph.min_content,
                    take(&mut self.current_line.min_content),
                );
```

Reply:
nice catch. I have updated it and also `self.paragraph.max_content`. Thanks!